demo.html

```diff
@@ -4,10 +4,15 @@
 <head>
   <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
   <title>Google Map demo</title>
-  <script src="../webcomponentsjs/webcomponents.min.js"></script>
+  <script src="../webcomponentsjs/webcomponents.js"></script>
   <link rel="import" href="google-map.html">
+  <link rel="import" href="google-map-marker.html">
```

Comment:
Is there a reason you broke this out? google-map-marker.html is only useful as a distributed node to google-map. I still think keeping them together makes sense.